.github/workflows/build-boxkit.yml

```diff
@@ -82,7 +82,13 @@ jobs:
           registry: ghcr.io
           username: ${{ github.actor }}
           password: ${{ secrets.GITHUB_TOKEN }}
-            
+        
+      # Install and check cosign   
+      - name: Install cosign
+        uses: sigstore/cosign-installer@v3.7.0
+      - name: Check cosign install
+        run: cosign version
+
       # Sign container
       - uses: sigstore/cosign-installer@v3.7.0
 
```

Comment:
We'd want to replace this with what you're PRing in I think?

Reply:
[This](https://github.com/faeizmahrus/distrobox-images-boxkit/actions/runs/12441987741/job/34739573741) happens on my fork of boxkit (the one my PRs are based on) when `cosign` isn't installed.

```
Run cosign sign -y --key env://COSIGN_PRIVATE_KEY ghcr.io/faeizmahrus/winebox-fedora@${TAGS}
/home/runner/work/_temp/1af756e3-4bfc-4756-a969-39efcf2ddc7e.sh: line 1: cosign: command not found
Error: Process completed with exit code 127.
```

Issue is [resolved](https://github.com/faeizmahrus/distrobox-images-boxkit/actions/runs/12442177918/job/34740000557) after installing `cosign`

Reply:
Seems like I accidentally removed the `cosign-installer` action on my fork
Sorry for the redundant PR.